sdk/core/azure-servicemanagement-legacy/azure/servicemanagement/_common_conversion.py

```diff
@@ -1,4 +1,4 @@
-﻿#-------------------------------------------------------------------------
+﻿# -------------------------------------------------------------------------
```

Comment:
@atombrella - This package has been deprecated. Please remove all changes made here. Thanks!